supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesJobPaths.scala

```diff
@@ -42,6 +43,13 @@ case class TesJobPaths private[tes] (override val workflowPaths: TesWorkflowPath
     case anyOtherPath: Path => anyOtherPath.pathAsString
   }
 
+  // Like above: Nothing should rely on these files existing, since only the Azure TES implementation will actually create them.
+  // Used to send the Azure TES log paths to the frontend.
+  val azureLogPathsForMetadata : Map[String, Any] = Map(
+    "tes_stdout" -> taskFullPath./("stdout.txt").pathAsString,
+    "tes_stderr" -> taskFullPath./("stderr.txt").pathAsString
```

Comment:
I'm wondering if the naming should reflect the purpose (frontend) but could use some context. Maybe let's sync with faces so I can get context quickly and contain the length of the semantics discussion 🙂

Semi-related, we have a bunch of metadata keys in one place [here](https://github.com/broadinstitute/cromwell/blob/7ab5582da138969126e7eb5c7267dbc66a8a78a8/services/src/main/scala/cromwell/services/metadata/CallMetadataKeys.scala#L5-L28), it's unclear whether this backend-specific one actually belongs in the main list though.

Reply:
Tangentially related comment: There's a method called `queryLogs` within `MetadataDatabaseAccess.scala` that currently looks for logs with specific keys (specifically "stdout" and "stderr" as two of them). The method basically queries for log-related metadata and processes that for the JSON response.

With two new keys we'd need to update these types of fetches (or any method that might use `CallMetadataKeys` to paint the metadata response) to account for these new potential keys. That said, its best to tackle these updates in another ticket.